utils/coffea/job_submission_helper.py

```diff
@@ -57,6 +57,30 @@ def __get_client(executor_name, n_workers, cores, memory, disk, time, partition,
             job_script_prologue=job_script_prologue,
         )
 
+    elif "ETP" in executor_name:
+        repo_directory = os.environ["SVJ_PROCESSING_ROOT"]
+        
+        cluster = HTCondorCluster(
+            scheduler_options={
+                "port": "3719",
```

Comment:
Does the port need to fixed at your uni cluster?
Otherwise, what about using the port argument to set the port number?

Reply:
yes, the port needs to be fixed (it actually has to be manually opened by admins)
I changed it to use the port argument now